practice-two/src/js/controllers/product.controller.js

```diff
@@ -0,0 +1,18 @@
+import HttpService from "../services/httpClient.service";
+import ProductView from "../views/product.view";
+import Product from "../models/product.model";
+
+class ProductController {
+  constructor() {
+    this.HttpService = new HttpService();
+    this.productView = new ProductView();
+  }
+
+  async showProduct() {
+    const products = await this.HttpService.get('product')
```

Comment:
Fetching products should be in models/, controller will call model to get data and update view